docs/structured-streaming-kafka-integration.md

```diff
@@ -388,6 +388,19 @@ The following configurations are optional:
   <td>streaming and batch</td>
   <td>Rate limit on maximum number of offsets processed per trigger interval. The specified total number of offsets will be proportionally split across topicPartitions of different volume.</td>
 </tr>
+<tr>
+  <td>minPartitions</td>
+  <td>int</td>
+  <td>0 (disabled)</td>
```

Comment:
Thank you for your first contribution, @arunpandianp .
However, this is wrong because this will mislead the users to try to set `0` and face `IllegalArgumentException`.
Technically, the default value is `None`. Just leave this line as a blank like `<td></td>`.